tests/integration-tests/tests/iam_policies/test_iam_policies.py

```diff
@@ -1,54 +0,0 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
It would be good to see this file as modified file instead of removed.

Reply:
I tried refactor it again. But it seems github only recognize move without change of the content